benchmark-overhead/src/test/java/io/opentelemetry/agents/LatestAgentSnapshotResolver.java

```diff
@@ -49,11 +49,12 @@ private String fetchLatestFilename(String version) throws IOException {
         .filter(elem -> {
           String classifier = $(elem).child("classifier").content();
           String extension = $(elem).child("extension").content();
+          // TODO (trask) this needs to be updated now that no more "all" artifact
           return "all".equals(classifier) && "jar".equals(extension);
```

Comment:
@breedx-splk any idea with this change if the classifier check should be changed to `""` or `null`?

Reply:
Is the idea here that we'll be publishing the full agent with exporters now without the `-all` classifier? If so, then I think we should just remove the classifier check entirely.

It looks like you're also publishing `-all` for backwards compat? Assuming that's temporary, yeah, I think we'll want to remove this classifier check...but it might be nice to see what the xml looks like after the first publish to confirm.

Reply:
the `-all` artifact will be attached to the github release for now for backwards compatibility, but won't be published.

there will be a new artifact `-slim` published after this tho

> it might be nice to see what the xml looks like after the first publish to confirm

this works for me 👍